<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Serial</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>串口名：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_serialName"/>
         </item>
        </layout>
       </item>
       <item row="0" column="3" colspan="2">
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>停止位：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_stopBit">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>-1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" colspan="2">
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>校验位：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_checkData">
           <item>
            <property name="text">
             <string>NoParity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>EvenParity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OddParity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SpaceParity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MarkParity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>-1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_9">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>波特率：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_baudRate">
           <property name="text">
            <string>115200</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" colspan="2">
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>数据位：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_dataBit">
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>-1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="QPushButton" name="pushButton_serial_open">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="2">
        <widget class="QPushButton" name="pushButton_serial_save">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QFormLayout" name="formLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>输入：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_serial_input"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QPushButton" name="pushButton_serial_send">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>接收</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit_sub"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>发送</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit_send"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
